lib/puma/plugin/statsd.rb

```diff
@@ -13,14 +13,16 @@ class StatsdConnector
   def initialize
     @host = ENV.fetch(ENV_NAME, nil)
     @port = ENV.fetch("STATSD_PORT", 8125)
+    @metric_prefix = ENV.fetch("STATSD_METRIC_PREFIX", nil)
+    @metric_prefix += ":" if @metric_prefix && !@metric_prefix.end_with?(":")
```

Comment:
I think the `:` delimiter has special meaning in the statsd message format.

Would it work to use `.` as the delimiter?

Reply:
You're right, I somehow messed it up.  Should be fixed now.